### ISSUE TITLE
Helm support

### ISSUE THREAD

**Issue by darrenhaken** [tinselspoon/intellij-kubernetes#9]
Any interest in also adding Helm support to this? It would be amazing.

**Comment by tinselspoon** [comment]
Thanks for the suggestion! Do you mind elaborating slightly on what features you'd like to see? Was it support for completion in the Helm-specific files like `Chart.yaml` and `requirements.yaml`?

**Comment by darrenhaken** [comment]
Yes exactly helm specific files and also it’s templating syntax like {{.Values.foo}} and {{template }}.

There’s something similar in VSCode which also does autocompletion and validation for both Helm and K8: https://marketplace.visualstudio.com/items?itemName=technosophos.vscode-helm

Do you think this is something of interest? I also don’t mind lending a hand if that helps.

**Comment by tinselspoon** [comment]
Thanks for that link; this definitely seems like it would be good to support. I'll have a look into how it could fit in this weekend.

**Comment by darrenhaken** [comment]
Thanks, let me know if you need any help.
